File: 412. Fizz Buzz/412.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> ans;
        string temp;
        for(int i=1; i<=n; ++i){
              temp = (i%3!=0 && i%5!=0) ? to_string(i) :
                     ((i%3==0 && i%5!=0) ? string("Fizz") :
                     (((i%3!=0 && i%5==0) ? string("Buzz") : string("FizzBuzz"))));
            ans.push_back(temp);
        }
        return ans;
    }
    
};

File: 0412. Fizz Buzz/412.py
class Solution(object):

    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        return [str(i) if (i % 3 != 0 and i % 5 != 0)
                else 'Fizz' if (i % 3 == 0 and i % 5 != 0)
                else 'Buzz' if (i % 3 != 0 and i % 5 == 0)
                else 'FizzBuzz' for i in range(1, n + 1)]
